### ISSUE TITLE
Upgrade OpenShift runtime to use latest Java LTS version (11)

### ISSUE THREAD

**Issue by pmonks** [symphonyoss/bot-github-chatops#51]
### Description of Problem:
With the recent release of the next Long Term Support (LTS) version of Java (version 11), the Foundation's OpenShift images should be upgraded to use it.  With this release, Java 8 (which the OpenShift images currently use) is no longer supported.

**Comment by maoo** [comment]
Thanks for raising this @pmonks .

As we use official centos distributions for our images, the end of support for OpenJDK 1.8 is June 2023; Red Hat is working on adding Java 11 in their official repositories, we will update the image as soon as they are released into the RHEL repositories (couldn't find a timeline yet).

**Comment by pmonks** [comment]
@maoo any updates on upgrading the base OpenShift image to the latest Java LTS (v11)?

**Comment by maoo** [comment]
Yes! I've updated the `latest` tag of our s2i-java-binary image, see https://github.com/maoo/s2i-java-binary/commit/74fba5090f6f798262fb385d239fb8722f40cdb7

I'll run some deployment tests to see if everything works as expected.
